docs/topics/http/decorators.txt

```diff
@@ -119,5 +119,5 @@ client-side caching.
 .. function:: never_cache(view_func)
 
     This decorator adds a ``Cache-Control: max-age=0, no-cache, no-store,
-    must-revalidate`` header to a response to indicate that a page should never
+    must-revalidate, private`` header to a response to indicate that a page should never
```

Comment:
Please wrap at 79 chars and add `versionchanged`, e.g.

```
    .. versionchanged:: 3.0

        ``private`` directive was added.
```

Reply:
Thanks for the comment!
I'll fix it.